.clang-format

```diff
@@ -15,6 +15,7 @@ BinPackParameters: false
 IndentCaseLabels: true
 BreakBeforeTernaryOperators: true
 BreakBeforeBinaryOperators: All
+BreakBeforeBraces: Allman
```

Comment:
I applied this 'braces' to `az_json_read.c`. It was 456 lines, now it's 533. IMHO, `Allman` style is not very good for a lot of small functions and code blocks but it's good for huge unreadable C-style 
functions :-)